includes/integrations/maxmind-geolocation/views/html-admin-options.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Admin View: Page - Admin options.
+ *
+ * @package WooCommerce\Integrations
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+?>
+
+<table class="form-table">
+	<tr valign="top">
+		<th scope="row" class="titledesc">
+			<label><?php esc_html_e( 'Database File Path', 'woocommerce' ); ?></label>
+		</th>
+		<td class="forminp">
+			<fieldset>
+				<legend class="screen-reader-text"><span><?php esc_html_e( 'Database File Path', 'woocommerce' ); ?></span></legend>
+				<input class="input-text regular-input" type="text" value="<?php echo esc_attr( $this->database_service->get_database_path() ); ?>" readonly>
```

Comment:
In my test, we are showing this path even if the file doesn't exist (before the user enters the key). Should we hide this field when there is no key or if the file doesn't exist? Or maybe change the description below that implies that the file was already downloaded?

Reply:
I've added some clarity to the description.

Reply:
The idea is to show where the database should be located, this replaces the system status note about it.
Also allows users to download and insert the database in the correct place if they like.